src/lib/server/database/index.ts

```diff
@@ -38,6 +40,23 @@ function updateLastConfession(db: Interface, guildId: Snowflake) {
         .returning({ confessionId: schema.guild.lastConfessionId });
 }
 
+async function insertAttachmentData(db: Interface, attachment: Attachment) {
+    const [result, ...otherResults] = await db
+        .insert(schema.attachment)
+        .values({
+            id: attachment.id,
+            filename: attachment.filename,
+            contentType: attachment.content_type,
+            url: attachment.url,
+            proxyUrl: attachment.proxy_url,
+        })
+        .returning();
+    strictEqual(otherResults.length, 0);
+    assert(typeof result !== 'undefined');
+    assert(typeof result.id === 'bigint');
+    return { attachmentId: BigInt(result.id) };
```

Comment:
Since we're only returning a single field here, I think it's more ergonomic to just return the ID straight-up.
```suggestion
    return BigInt(result.id);
```